tasks/fix-cat2.yml

```diff
@@ -942,9 +942,9 @@
 - name: "MEDIUM | RHEL-07-021030 | PATCH | All world-writable directories must be group-owned by root, sys, bin, or an application group."
   block:
     - name: "MEDIUM | RHEL-07-021030 | AUDIT | All world-writable directories must be group-owned by root, sys, bin, or an application group."
-      command: find {{ rhel7stig_local_mounts | join(' ') }} -type d -perm -002 -gid +999
+      command: find {{ rhel7stig_local_mounts | join(' ') }} -xdev -type d -perm -002 -gid +999
       changed_when: no
-      register: rhel_07_021030_audit
+      register: rhel_07_021030_audit.stdout != ""
```

Comment:
that should be on the "changed_when" line above, but I suppose the find command would also have to exclude items group-owned by root

Reply:
The find command's `-gid +999` should exclude any files owned by system-level users, including root, right?

Reply:
You are probably correct.  I didn't test it.  In that case, it is correct to make:
```
changed_when: rhel_07_021030_audit.stdout != ""
```

and restore this line to:
```
register: rhel_07_021030_audit
```